international-labour-organization.csl

```diff
@@ -109,15 +113,15 @@
     </choose>
   </macro>
   <macro name="pages">
-    <group>
-      <label variable="page" form="short" suffix=" "/>
+    <group prefix="p">
+      <label suffix=" " variable="page" form="short"/>
       <text variable="page"/>
     </group>
   </macro>
   <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
     <layout prefix="(" suffix=")" delimiter="; ">
       <group delimiter=" ">
-        <text macro="author-short"/>
```

Comment:
Your style doesn't validate anymore as you're not using the author-short macro anymore. Something tells me that this might not be what you want.
Could you also update the documentation links?